TFNetworkLayer.py

```diff
@@ -2961,11 +2961,10 @@ def _get_out_type_from_opts(cls, n_out, filter_size, padding, strides=1, dilatio
                               auto_use_channel_first=False,
                               **kwargs):
     data = get_concat_sources_data_template(sources)
-    # The output format is the same as the input.
-    if data.feature_dim_axis == 1:
-      shape = [n_out] + [None] * len(filter_size)
-    else:
-      shape = [None] * len(filter_size) + [n_out]
+    auto_use_channel_first = auto_use_channel_first or data.is_batch_feature_major
+    # The output format is the same as the input for now.
```

Comment:
Why "for now"? Remove that. Maybe write "we maybe change to channel-first below".